.github/workflows/test.yml

```diff
@@ -0,0 +1,122 @@
+---
+name: Test
+
+on:
+  pull_request:
+  push:
+
+jobs:
+
+  lint_and_validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: manics/action-k3s-helm@v0.2.1
```

Comment:
Will this mean we start an entire k3s cluster just to install Helm 3? If so I favor retaining setup_helm in ci/common and let run that here instead.

Reply:
Will do....